yaccparser.py

```diff
@@ -711,6 +713,47 @@ def p_multiplyinstruction(p):
     # so we just pack it in a bytes object
     p[0] = (struct.pack("<I", p[0]), None)
 
+def p_multiplylonginstruction(p):
+    """multiplylonginstruction : OPMULL logmnemonic flagscondandspace REG COMMA REG COMMA REG COMMA REG
+                               | OPMULL logmnemonic flagscondandspace REG COMMA REG COMMA SHARP CONST"""
```

Comment:
There a missing "COMMA REG" pair here, since this instruction requires 4 arguments